bundles/org.openhab.ui/web/src/pages/settings/items/item-edit.vue

```diff
@@ -2,11 +2,11 @@
   <f7-page @page:afterin="onPageAfterIn">
     <f7-navbar :title="createMode ? 'Create New Item': (editable ? 'Edit Item' : 'Item Details')" :back-link="editable ? 'Cancel': 'Back'">
       <f7-nav-right>
```

Comment:
We probably should add a `v-if="ready"` here, because currently the lock is displayed for a short period of time.

Reply:
Good catch!

Reply:
oops I used v-show. Should I change it to v-if?